### PR TITLE
Updated Librespeed Docs

### PR DESCRIPTION
**Pull request by HarleyL7**

### CODE REVIEW

File: docs/applications/librespeed.mdx [hunk 1]
```diff
@@ -6,6 +6,36 @@ sidebar_label: Librespeed
...
+Once setup, Librespeed will be available at the link `https://<hostname.ltd>/librespeed`
+
+## Service Management
+
+Like all box configured applications, you can manage Librespeed via SSH with box with start, stop, restart, enable and disable commands.
+
+import SystemdTabs from '../snippets/systemdtabs.mdx';
+
+<SystemdTabs service="librespeed"/>
+
```
Comment (liaralabs) [inline docs/applications/librespeed.mdx:30]: There is no systemd service for librespeed, it's just a page served through nginx 😉

Comment (brettpetch) [reply docs/applications/librespeed.mdx:30]: 🥸

Comment (HarleyL7) [reply docs/applications/librespeed.mdx:30]: 🤡 I'll fix this tomorrow lol

**Comment by liaralabs** [comment]
Don't forget to update the sidebar as well:

https://github.com/liaralabs/docs.swizzin.ltd/blob/master/sidebars.js

**Comment by brettpetch** [comment]
> 
> 
> Don't forget to update the sidebar as well:
> 
> https://github.com/liaralabs/docs.swizzin.ltd/blob/master/sidebars.js

doesn't it already exist in the sidebar under Utilities?

**Review by HarleyL7 — COMMENTED** [review]
Removed Service management

**Pull request merged**